backend/plugin/parser/differ/mysql/constraint_test.go

```diff
@@ -238,8 +238,8 @@ func TestIndexType(t *testing.T) {
 		{
 			old: `CREATE TABLE book(name VARCHAR(50) NOT NULL, INDEX book_idx USING BTREE(name));`,
 			new: `CREATE TABLE book(name VARCHAR(50) NOT NULL, INDEX book_idx USING HASH(name));`,
-			want: "ALTER TABLE `book` DROP INDEX `book_idx`;\n\n" +
-				"ALTER TABLE `book` ADD INDEX `book_idx` (`name`) USING HASH;\n\n",
+			want: "DROP INDEX `book_idx` ON `book`;\n\n" +
```

Comment:
Just for curiorsity, why we using `DROP INDEX ON` statement instead of `ALTER TABLE DROP INDEX` statement?

Reply:
For MySQL, there is no difference between using DROP INDEX and ALTER TABLE DROP INDEX statements. 
However, in SDL, we only allow users to utilize the CREATE INDEX statement. 
In order to maintain a uniform style, I have opted to use the DROP INDEX statement instead of ALTER TABLE DROP INDEX.

Reply:
SGTM